src/js/utils/Units.js

```diff
@@ -55,9 +54,7 @@ const Units = {
       return amount;
     }
 
-    options = _.extend({
-      decimalPlaces: 2
-    }, options);
+    options = Object.assign({decimalPlaces: 2}, options || {});
```

Comment:
We can default assign `options` in the argument with `{}`, if we are only handling undefined here